packages/phone-number-privacy/signer/src/signing/get-partial-signature.ts

```diff
@@ -106,7 +125,8 @@ export async function handleGetBlindedMessagePartialSig(
     } else {
       signMessageResponse = signMessageResponseSuccess
     }
-    response.json(signMessageResponse)
+    logger.debug('Signature retrieval success')
+    response.status(200).json(signMessageResponse)
```

Comment:
wasn't this the default without the `status(200)`?

Reply:
yes, was just trying to be explicit since we are populating the error field but am happy to switch it back :)